README.md

```diff
@@ -26,6 +26,24 @@ OSU CSE 5911 Capstone Project: Elephant Vending Machine in coordination with Cin
 1. `pip install -e .`
 1. `flask run`
 
+## Configuring Remote Pis for GPIO Pin Reading
+1. Ensure Pis are running Raspbian
+1. Open the Raspberry Pi Configuration Tool
+1. Set "Remote GPIO" to "enabled"
+1. Configure this to run at boot in the future
+    * Using a terminal, run the following `sudo systemctl enable pigpiod`
+    * Now to manually enable remote access at this time, run `sudo systemctl start pigpiod`
+1. For more information, visit [this documentation](https://gpiozero.readthedocs.io/en/stable/remote_gpio.html)
```

Comment:
Is this section still necessary now that we're using a single inverter board? We may be able to remove the entire gpiozero dependency.

Reply:
Fair point, I'll remove the dependency and this section of the README.